web/src/containers/viewOnly/ApdStateProfile.js

```diff
@@ -49,31 +49,38 @@ const ApdStateProfile = ({ stateProfile, keyPersonnel }) => {
     );
   };
 
+  const MedicaidOffice = ({ medicaidOffice }) => {
+    const {address1, address2, city, state, zip } = medicaidOffice;
+
+    // Since we provide a default State don't check if falsy
+    if(!address1 && !address2 && !city && !zip) {
+      return ( <span>No reponse was provided</span> ) ;
+    }
+
+    return (
+      <address>
+        {address1}
+        <br />
+        {address2 &&
```

Comment:
Can you put a !! in front of the first address2? I see what you are doing here, but it's a little confusing at first glance. I don't want someone to mistake it for a typo.

https://dev.to/sanchithasr/what-is-the-double-bang-operator-in-javascript-4i3h